ui/app/components/dropdowns/components/account-dropdowns.js

```diff
@@ -251,7 +262,10 @@ class AccountDropdowns extends Component {
               this.props.actions.showAccountDetailModal()
             },
             style: Object.assign(
-              {},
+              {
```

Comment:
Perhaps a `const dropdownMenuItemStyle` like you did in network-dropdown would be useful here.